src/__tests__/ID3v2FrameReader-test.js

```diff
@@ -87,80 +87,199 @@ describe("ID3v2FrameReader ", function() {
   });
 
   describe("T* text tags", function() {
-    it("should read text with iso-8859-1 charset", function() {
+    describe("T000 - TZZZ, excluding TXXX", function() {
       var frameReader = ID3v2FrameReader.getFrameReaderFunction("T*");
-
-      expect(frameReader).toBeDefined();
-
-      var fileData = [].concat(
-        [0x00], // encoding
-        [0xe3]
-      );
-      var fileReader = new ArrayFileReader(fileData);
-      var data = frameReader(0, fileData.length, fileReader);
-
-      expect(data).toEqual("ã");
+      var expected = "ã";
+
+      it("asserts the function is defined", function() {
+        expect(frameReader).toBeDefined();
+      });
+
+      it("should read text with iso-8859-1 charset", function() {
```

Comment:
These are just put in their own `describe` block, along with removing some of the `[].concat(...)` calls for all the `var fileData` declarations.